.github/workflows/push-build.yaml

```diff
@@ -78,13 +78,7 @@ jobs:
 
       - name: consider debugging
         if: failure()
-        run: |
-          # Enable tmate only in the Rook fork, where the USE_TMATE secret is set in the repo, or if the action is re-run
-          if [ "$GITHUB_REPOSITORY_OWNER" = "rook" ] || [ -n "${{ secrets.USE_TMATE }}" ] || [ "$GITHUB_RUN_ATTEMPT" -gt 1 ]; then
-            echo USE_TMATE=1 >> $GITHUB_ENV
-          fi
-
-      - name: set up tmate session for debugging
-        if: failure() && env.USE_TMATE
-        uses: mxschmitt/action-tmate@v3
         timeout-minutes: 60
```

Comment:
can you also include this in the test `timeout-minutes: 60` mostly it is common

Reply:
This is still to be implemented in composite action, they don't support timeout-minutes currently, i mentioned it in commit msg, once the support comes out, i will update the code for the same, thanks

Reply:
okay thanks